From 57d0ca75c69a12e974f664e99675a95ecc618276 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Tue, 11 Aug 2020 09:37:17 +0200
Subject: [PATCH 23/34] arch/arm/dts/Makefile: Fix topic board errors

Add missing xdp dtb
Fix typos "dts" instead of "dtb"
---
 arch/arm/dts/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index c3e87486b9..12dcf8aece 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -362,8 +362,9 @@ dtb-$(CONFIG_ARCH_ZYNQMP) += \
 	zynqmp-sck-kv-g-revA-pl.dtbo		\
 	zynqmp-topic-miamimp.dtb		\
 	zynqmp-topic-miamimp-xilinx-xdp-v1r1.dtb	\
-	zynqmp-topic-tdpzu9.dts			\
-	zynqmp-topic-ttpzu9.dts			\
+	zynqmp-topic-miamimp-xilinx-xdp.dtb	\
+	zynqmp-topic-tdpzu9.dtb			\
+	zynqmp-topic-ttpzu9.dtb			\
 	zynqmp-vpk120-revA.dtb			\
 	zynqmp-vp-x-a2785-00-revA.dtb		\
 	zynqmp-zcu100-revC.dtb			\
-- 
2.17.1

